.claude/scripts/effective_env_lookup.py

```
@@ -0,0 +1,306 @@
+    """Run effective_env.py for one config file and parse JSON output."""
+    cmd = [
+        sys.executable,
+        str(script),
+        str(config_file),
+        job_name,
+        "--repo-root",
+        str(repo_root),
+    ]
+    if filter_str:
+        cmd.extend(["--filter", filter_str])
+    proc = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.15.18)

[error] `subprocess` call: check for execution of untrusted input

(S603)